scripts/setup/config-create.sh

```diff
@@ -25,101 +25,93 @@ PROJECT=""
 ZONE="europe-west4-b"
 CLUSTER_NAME="kctf-cluster"
 DOMAIN_NAME=""
-
-config=""
-read_config() {
-    read -e -p "  $2: " -i "${!1}" "$1"
-    line="${1}=${!1}"
-    config="${config}"$'\n'"${line}"
-}
+START_CLUSTER="0"
 
 if [ -d ${CONFIG_DIR}/challenges ]; then
     load_chal_dir
-    echo
-    echo "Configuring cluster for challenge directory ${CHAL_DIR}"
-else
-    echo
-    read -e -p "In which directory will challenges be stored?: " "CHAL_DIR"
-
-    if [[ $CHAL_DIR == ~\/* ]]; then
-        CHAL_DIR="${HOME}/${CHAL_DIR:2}"
-    fi
+fi
 
-    "${DIR}/scripts/setup/challenge-directory.sh" "${CHAL_DIR}"
+function usage {
+    echo "$(basename $0) [args]" >&2
+    echo -e "\t--chal-dir\tdirectory in which challenges are stored (default: \"${CHAL_DIR}\")" >&2
```

Comment:
What happens if chal dir is not defined?

Reply:
it will just print default: ""